### PR TITLE
Add summits section

### PR DESCRIPTION
**Pull request by AndreiLobanovich**

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ import Community from '@/components/community';
 import Footer from '@/components/footer';
 import Navigation from '@/components/navigation';
 import Callout from '@/components/callout';
-import { createMetadata } from '@/lib/metadata';
-import type { Metadata } from 'next';
-import type { ReactElement } from 'react';
```
Comment (chrisgervang) [inline app/page.tsx:12]: Can we avoid lint opinion changes?

Comment (AndreiLobanovich) [reply app/page.tsx:12]: Sure

**Comment by chrisgervang** [comment]
Hey @AndreiLobanovich, I was wondering if you could take a look at this. Thank you!

File: customtypes/home/index.json [hunk 2]
```diff
@@ -262,6 +334,97 @@
...
+          }
+        }
+      },
+      "slices7": {
+        "type": "Slices",
+        "fieldset": "Slice Zone",
+        "config": {
+          "choices": {}
+        }
+      }
```
[the remark this answers is not in the document]
Comment (AndreiLobanovich) [reply customtypes/home/index.json]: Hey, @chrisgervang, sorry, seems like i've added this slice by accidend while i was figuring out prismic.

**Comment by chrisgervang** [comment]
Thanks for removing that. Can you please add some styles for mobile?

image
image

**Review by chrisgervang — APPROVED** [review]
Looks great, thank you for your contributions!

Can I make some edits to the logo bubbles in Prismic?

**Comment by AndreiLobanovich** [comment]
> Looks great, thank you for your contributions!
> 
> Can I make some edits to the logo bubbles in Prismic?

Sure thing

**Pull request merged**